app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/topBar/changePasswordModal.tsx

```diff
@@ -0,0 +1,126 @@
+/** @file Managing the logic and displaying the UI for the password change function. */
+import React from 'react'
+import * as auth from '../../../authentication/providers/auth'
+import * as hooks from '../../../hooks'
+import * as utils from '../../../utils'
+import * as Icons from '../../../components/svg'
+import toast from 'react-hot-toast'
+import Modal from '../../../components/model'
+
+interface ResetPasswordModalProps {
+    visible: boolean
+    handleCancel: () => void
+}
+// ==========================
+// === ResetPasswordModal ===
+// ==========================
+
+const ChangePasswordModal: React.FC<ResetPasswordModalProps> = props => {
```

Comment:
prefer avoiding `React.FC` - just use a regular function
(see [here](https://github.com/facebook/create-react-app/pull/8177) for more info)